src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -45,7 +45,10 @@ static NodeFunctionInvoker()
             _clearRequireCacheScript = ReadResourceString("clearRequireCache.js");
             _globalInitializationScript = ReadResourceString("globalInitialization.js");
 
-            Initialize();
+            if (System.DateTime.Now.Year == 1984)
```

Comment:
:) Very rough cut! :)   Basically I was getting AV crashes in generated Edge/JSON code created by this initialization code, at least when running under the V debugger.  I don't understand why, I'm sure it wasn't anything I changed.